03_how_ln_works.asciidoc

```diff
@@ -723,6 +723,8 @@ When settled on-chain, it must be settled in "satoshis".
 
 Both the Bitcoin network and the Lightning network use the same monetary units: bitcoins. 
 Lightning payments use the very same bitcoins as Bitcoin transactions.
+As an implication, because the monetary unit is the same, also the economic monetary limit is the same: 21 million bitcoins.
+From these 21 million bitcoins, some are locked on the Bitcoin blockchain so that they can be used on Lightning.
```

Comment:
I would argue all bitcoins are always locked on the bitcoin blockchain. 

I think the more precise statement would be. "some are locked in 2 out of 2 multisig addresses, so that they can empower / enable a payment channel on the Lightning Network"

Reply:
Adjusted to reflect your suggestions in commit #2 
a99bcf77bb088f045da575d5d96edd84b3d58997